packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -468,10 +515,25 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
       colormap,
       preset,
       interpolationType,
+      slabThickness,
```

Comment:
I don't think this belongs here, slabThickness is not a property of viewport I believe

Reply:
Yeah sure it is fine